src/main/resources/templates/index.html

```diff
@@ -77,6 +79,10 @@
                                 <option th:each="i : ${#numbers.sequence(1, book.getInventory())}" th:text="${i}" />
                             </select>
                             <button class="ease-in-out duration-300 hover:bg-blue-800 hover:text-white border-blue-800 border-2 rounded-md px-4 py-1 ml-2">Add to cart</button>
+                            <form action="/home/transit-to-edit-book" method="GET">
+                                <input name="id" type="hidden"th:value="${book.getId()}">
+                                <button class="ease-in-out duration-300 hover:bg-blue-800 hover:text-white border-blue-800 border-2 rounded-md px-4 py-1 ml-2">Edit book</button>
```

Comment:
While you're in here could you add a delete book button as well please?

Reply:
I added the delete button, but I made a couple of changing to the mapping due to thymeleaf not supporting PUT and DELETE method. Overall, I won't affect the program